ed25519/tests/ed25519.rs

```diff
@@ -0,0 +1,45 @@
+
+#[test]
+fn run_wycheproof() {
+    for test_name in wycheproof::eddsa::TestName::all() {
+        if !matches!(test_name, wycheproof::eddsa::TestName::Ed25519) {
+            continue;
+        }
+
+        let test_set = wycheproof::eddsa::TestSet::load(test_name)
+            .expect("error loading wycheproof test for name {test_name}");
+
+        println!("Test Set {test_name:?}");
+        for (k, test_group) in test_set.test_groups.into_iter().enumerate() {
+            let pk = &test_group.key.pk;
+            if pk.len() != 32 {
+                println!("Skipping test group {k}: private key length {} != 32", pk.len());
+                continue;
+            }
+            let pk_buf: [u8; 32] = pk.as_slice().try_into().unwrap();
+
+            for (i, test) in test_group.tests.into_iter().enumerate() {
+                let comment = &test.comment;
+                println!("Test {i}: {comment}");
+
+                let sig_len = test.sig.len();
+                if sig_len != 64 {
```

Comment:
```suggestion
                if sig_len != 64  && test.result == wycheproof::TestResult::Invalid {
```
Just to make sure that in the skipped tests the Wycheproof suite expects us to error, which seems to be the case.

Reply:
Thanks! I updated the code to reflect this.